pdpserver/server/log_mgr.go

```diff
@@ -0,0 +1,19 @@
+package server
+
+import (
+	"github.com/Sirupsen/logrus"
+)
+
+type LevelCheckable interface {
```

Comment:
All exported symbols which names start with capital letter should have documentation comments. You can run `golint` or `golint -min_confidence 0` to see if your code has that kind of issues.

Reply:
Additionally, it would be good to mention this feature somewhere in README. But for now we seem haven't created section in README for embedding pdpserver as a library.